packages/eas-cli/src/commands/branch/delete.ts

```diff
@@ -0,0 +1,180 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import { DeleteUpdateBranchResult, Update } from '../../graphql/generated';
+import Log from '../../log';
+import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
+import { findProjectRootAsync, getProjectAccountName } from '../../project/projectUtils';
+import { promptAsync, toggleConfirmAsync } from '../../prompts';
+import { ensureLoggedInAsync } from '../../user/actions';
+
+async function getBranchInfoAsync({
+  appId,
+  name,
+}: {
+  appId: string;
+  name: string;
+}): Promise<{ branchId: string }> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<
+        {
+          app: {
+            byId: {
+              updateBranchByName: {
+                id: string;
+                name: string;
+                updates: Pick<Update, 'id'>[];
+              };
+            };
+          };
+        },
+        {
+          appId: string;
+          name: string;
+        }
```

Comment:
```suggestion
        ViewBranchQuery,
        ViewBranchQueryVariables
```
`graphql-codegen` will generate these types automatically for you, no need to write and maintain them by hand.

Add this file to the paths of files with GraphQL queries and then re-run `yarn generate-graphql-code`. The config is here:
https://github.com/expo/eas-cli/blob/2e25ed81ca52d9d25afeadd751248d885ed152a9/packages/eas-cli/graphql-codegen.yml#L3-L5

Reply:
Nice!
I added the whole commands sub-directory:
`'src/commands/**/*.ts'`

Will go through and convert some more types in a follow up PR.